tests/_fixtures/healthy-tasks-in-mesos-and-marathon/groups.json

```diff
@@ -148,7 +130,7 @@
           ],
           "startedAt": "2017-01-09T20:02:26.435Z",
           "version": "2017-01-09T20:02:12.029Z",
-          "id": "confluent-kafka.825e1e2e-d6a6-11e6-a564-8605ecf0a9df",
+          "id": "confluent-kafka.instance-825e1e2e-d6a6-11e6-a564-8605ecf0a9df._app.1",
```

Comment:
@GeorgiSTodorov looks like this one was the missing bit

Reply:
Interesting. Thanks for fixing it.